truffle.js

```diff
@@ -38,7 +38,15 @@ module.exports = {
       gas: 0xfffffffffff,
       gasPrice: 0x01,
     },
-  }
+  },
+  compilers: {
```

Comment:
do we still need the top level `solc:` option?

Reply:
i think we may be able to get rid of that

Reply:
interesting, i know it picks up the compiler version, but maybe the solc is not somehow...or maybe its in your local truffle version not using it properly? Didn't you say you weren't on truffle 5 yet?

Reply:
we can also keep for backwards compatibility on older truffle versions